README.md

```diff
@@ -36,23 +36,22 @@ GitHub action needs the following credentials for running.
 
 Zulip API key is used for fetching the messages of public streams from the Zulip organization. We recommend creating a bot and using it's API key instead of using your own API key. See https://zulip.com/help/add-a-bot-or-integration for more details.
 
-#### GitHub Personal Access Token
+#### GitHub Token
 
-The token is used by the GitHub action for pushing to the repo and running GitHub page builds. You can generate the token by going to https://github.com/settings/tokens. Make sure to enable `repo` and `workflow` while generating the token.
+The token is used by the GitHub Actions for pushing to the repo and running GitHub page builds.
+It is generated automatically by GitHub Actions itself at `secrets.GITHUB_TOKEN` and can be used right away. This is the recommended way because it only provides access to the repo that is running the actions and not other repo on the account. See the [official documentation](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token) for further information. Alternatively, you can use GitHub Personal Access Token.
```

Comment:
I think the sentence starting with "this is the recommended way because ..." and the following sentences are superfluous. It's unnecessary to let users be aware of the not-recommended personal access token.

Reply:
The info on permission scope etc is unnecessary.